components/raftstore/src/store/peer.rs

```diff
@@ -2341,7 +2342,13 @@ where
                 return false;
             }
             Ok(RequestPolicy::ReadIndex) => return self.read_index(ctx, req, err_resp, cb),
-            Ok(RequestPolicy::ProposeNormal) => self.propose_normal(ctx, req),
+            Ok(RequestPolicy::ProposeNormal) => {
+                if disk::disk_full_precheck(0) || disk::is_disk_full() {
```

Comment:
Why 0 instead of store_id?

Reply:
It means here we no need to get the store id.